### DISCUSSION TITLE
QstashError: {"error":"invalid destination url: endpoint has invalid scheme, add http:// or https://"}

### DISCUSSION THREAD

**Discussion by Vuducvi1999** [upstash/product-roadmap#121] [Announcements]
Hi guys,
I am trying to shift from general qstash to workflow, but my first init continue to raise weird error.
![image](https://github.com/user-attachments/assets/afab6743-e293-4f34-a97b-fe846394707b)

Its seem like context.run is calling invalid endpoint
![image](https://github.com/user-attachments/assets/66543d72-2e58-41f4-b99e-8378922970f2)
context.run call to invalid scheme
![image](https://github.com/user-attachments/assets/4a0ee052-971c-4608-8449-798cc03f2657)
while the real one should be this
![image](https://github.com/user-attachments/assets/5a659eed-5dea-47ad-ab6e-49afdc0cdd79)

**Comment by sancar** [comment]
@Vuducvi1999 Your issue moved here -> https://github.com/upstash/workflow-js/issues/149
